migrations/2025-01-09-144233_no-image-token/down.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE local_image
+    ADD COLUMN pictrs_delete_token text NOT NULL DEFAULT '';
+
+ALTER TABLE local_image
+    ALTER COLUMN pictrs_delete_token DROP DEFAULT;
+
+ALTER TABLE local_image
+    ADD COLUMN published_new timestamp with time zone DEFAULT now() NOT NULL;
+
+UPDATE
+    local_image
+SET
+    published_new = published;
+
+ALTER TABLE local_image
+    DROP COLUMN published;
+
+ALTER TABLE local_image RENAME published_new TO published;
```

Comment:
I know this is the down migration, but why change every published column in the table to now? That might possibly mess up sorting (IE what order images got uploaded) if it needed to be run.

Reply:
All this is to restore the original column order and pass the diff check as discussed with dullbananas above. It shouldnt change any of the values.

Reply:
Ah gotcha.